buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/converter.py

```diff
@@ -45,4 +83,19 @@ def get_metadata(self):
 
         # Get today date
         metadata["NWBFile"]["session_start_time"] = date
+
+        # Add additional NWBFile metadata
+        # NOTE: experimenters is specifid in the metadata.yml file
+        metadata["NWBFile"]["session_id"] = session_mat["session"]["general"]["name"]
+        metadata["NWBFile"]["notes"] = session_mat["session"]["general"]["notes"]
```

Comment:
This appears to be producing the string `Notes:    Description from xml: ` which is not very useful

Is there a way to skip this addition to the `NWBFile` metadata if it has this form?